### PR TITLE
Improve cos validation error messages

### PR DESCRIPTION
**Pull request by jaywcarman**

### CODE REVIEW

File: app/models/manageiq/providers/ibm_cloud/object_storage/storage_manager.rb [hunk 1]
```diff
@@ -182,9 +191,10 @@ def self.verify_iam(crn, api_key)
 
     begin
       resource_controller_api.get_resource_instance(:id => crn).result
-    rescue IbmCloudResourceController::ApiException => err
-      _log.error("GUID resource lookup failed: #{err.code} #{err.error}")
-      raise MiqException::MiqInvalidCredentialsError, err.error
+    rescue Exception => err
+      error_message = "CRN resource lookup failed: #{err.code} #{err.error}"
```
Comment (agrare) [inline app/models/manageiq/providers/ibm_cloud/object_storage/storage_manager.rb:195]: If we are going to rescue the more general `Exception` class then we can't count on having `:code` or `:error`
When this happens what exception is raised?

**Comment by Fryguy** [comment]
If these error messages will bubble up to the user they should probably be i18n'd

File: app/models/manageiq/providers/ibm_cloud/object_storage/storage_manager.rb (hunk 1 above)
Comment (Fryguy) [reply app/models/manageiq/providers/ibm_cloud/object_storage/storage_manager.rb:195]: Oh yeah, this is kinda dangerous - is there something more specific we can capture?

Comment (jaywcarman) [reply app/models/manageiq/providers/ibm_cloud/object_storage/storage_manager.rb]: Should have used `IBMCloudSdkCore::ApiException` (force pushed a fix)

**Review by agrare — APPROVED** [review]

**Pull request merged**